### PR TITLE
Removes a bunch fo Spawns in Life

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Removes a whole bunch of (seemingly) unnecessary spawns in `Life`

### CODE REVIEW

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -126,8 +126,7 @@
 		if(isturf(loc))
 			I.remove(src)
 			I.forceMove(get_turf(src))
-			spawn()
-				I.throw_at(get_edge_target_turf(src,pick(alldirs)),rand(1,3),5)
+			I.throw_at(get_edge_target_turf(src,pick(alldirs)),rand(1,3),5)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/carbon.dm]: `throw_at` uses a system whereby it doesn't need to spawn anymore, at least, as far as I know @tigercat2000

File: code/modules/mob/living/carbon/human/life.dm [hunk 2]
```diff
@@ -1090,7 +1090,7 @@
 			// Humans can lack a mind datum, y'know
 			if(H.mind && (H.mind.assigned_role == "Detective" || H.mind.assigned_role == "Coroner"))
 				return //too cool for puke
-			to_chat(H, "<spawn class='warning'>You smell something foul...")
+			to_chat(H, "<span class='warning'>You smell something foul...")
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/life.dm]: spawn class.

The new text colorizer.

File: code/modules/mob/living/carbon/life.dm [hunk 3]
```diff
@@ -236,21 +235,18 @@
...
+	for(var/mob/living/M in stomach_contents)
+		if(M.loc != src)
+			stomach_contents.Remove(M)
+			continue
+		if(stat != DEAD)
+			if(M.stat == DEAD)
 				stomach_contents.Remove(M)
+				qdel(M)
 				continue
-			if(istype(M, /mob/living) && stat != 2)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/life.dm]: Why do an istype check on an typed for loop?

File: code/modules/mob/living/carbon/life.dm [hunk 4]
```diff
@@ -236,21 +235,18 @@
...
-				if(M.stat == 2)
-					M.death(1)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/life.dm]: If they're dead, call `death` on them again, for some reason??

File: code/modules/mob/living/carbon/life.dm [hunk 5]
```diff
@@ -236,21 +235,18 @@
 		wetlevel = max(wetlevel - 1,0)
 
 /mob/living/carbon/handle_stomach()
-	spawn(0)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/life.dm]: But why?

File: code/modules/mob/living/carbon/slime/slime.dm [hunk 6]
```diff
@@ -281,11 +281,9 @@
 				if(prob(90) && !client)
 					Discipline++
 
-				spawn()
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/slime/slime.dm]: Good heavens almighty is slime code AWFUL.

File: code/modules/mob/living/carbon/human/life.dm (hunk 2 above)
Comment (Crazylemon64) [inline code/modules/mob/living/carbon/human/life.dm]: The span isn't terminated

**Comment by alexkar598** [comment]
You spelled "fo" instead of "of" in the title

File: code/modules/mob/living/carbon/life.dm (hunk 3 above)
Comment (taukausanake) [inline code/modules/mob/living/carbon/life.dm]: Redundant redundancy?

File: code/modules/mob/living/carbon/life.dm (hunk 4 above)
Comment (taukausanake) [inline code/modules/mob/living/carbon/life.dm]: Make them super dead

Comment (Citinited) [inline code/modules/mob/living/carbon/life.dm]: Wait does this do a check for if someone's dead only on mobs that aren't dead? 😆

Comment (outbools) [inline code/modules/mob/living/carbon/life.dm]: That's Markolie's code from #67

Comment (taukausanake) [inline code/modules/mob/living/carbon/life.dm]: HOW OLD ARE YOU PEOPLE

**Pull request merged**